### ISSUE TITLE
Android support?

### ISSUE THREAD

**Issue by patoroco** [pompopo/PRealm#2]
I'm starting with react-native, and I would like to ask if you plan to support android in this library?

Thanks! :)

**Comment by pompopo** [comment]
Yes. I'm planning to support android, and considering how to implement that.
`javassist` looks like good to create `RealmObject` dynamically. http://jboss-javassist.github.io/javassist/ 
If you have an idea, please tell me.
Thanks
